crystal_toolkit/apps/examples/fermi_surface.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+import dash
+
+# dos and bs data from local jsons
+from monty.serialization import loadfn
```

Comment:
Iirc, `monty` is just a transient dep atm. Should we add it to `pyproject.toml` since it's used in a dozen other places?

Reply:
Honestly I don't think it makes much odds -- it's so baked into pymatgen, and pymatgen is such a core requirement, that I think we'd only explicitly need to list it if we had a different minimum version. But by all means add if you feel strongly!